### ISSUE TITLE
Requirements JavaScript

### ISSUE THREAD

**Issue by PixelmonPRO** [ustc-zzzz/VirtualChest#33]
Hello!) I try add specific requirements, but not work)
Example: 
```
    Slot0 = [{
        Item {
            Count = 1
            ItemType = "pixelmon:pixelmon_sprite"
            UnsafeData = {
                SpriteName = "pixelmon:sprites/pokemon/%trainer_team_1_nationalid%"
            }
            UnsafeDamage = 0
            DisplayName = "&4Slot &b#1"
            ItemLore = [
                "&eFirst Pokemon"
            ItemEnchantments = [
                "minecraft:infinity"
            ]
            HideEnchantments = true
        }
        PrimaryAction {
            Command = "command 1"
            KeepInventoryOpen = false
        }
        SecondaryAction = {
            Command = "command 1"
            KeepInventoryOpen = false
        }
        Requirements = "%trainer_team_1% =< 'null'"
    }, {
        Item {
            Count = 1
            ItemType = "minecraft:barrier"
            UnsafeDamage = 0
            DisplayName = "&4None Pokemon!"
        }
    }]
```
I try specify none name (null) in placeholder. Placeholder `%trainer_team_1_nationalid%
How correctly can this be indicated?)

**Comment by ustc-zzzz** [comment]
Maybe you should check the value of `%trainer_team_1%` before you decide what you should write. Besides, you should write `a != b` if you want check if `a` is not equal to` b`.

**Comment by PixelmonPRO** [comment]
Thanks for the answer! The update is hard to come by, I change a lot of configs ... I made a mistake in the first message, I'm sorry.
Tried as you wrote, but so far without result. Placeholder parse name Pokemon, example: %trainer_team_1% parse bulbasaur, but `Requirements = "%trainer_team_1% != 'null'"` not check if the Pokémon is not on the team...
I would like to know when will update the latest commits? Logs in my servers for the last day, exceeded ower 10gb :) Spawm JS...
Sorry for my english)
